### PR TITLE
Syntax changes and model update changes

### PR DESCRIPTION
**Pull request by michplunkett**
## Description of Changes
A lot of small syntax changes along with adding missing field inclusions.

## Tests and linting
 - [x] This branch is up-to-date with the `develop` branch.
 - [x] `pytest` passes on my local development environment.
 - [x] `pre-commit` passes on my local development environment.

### CODE REVIEW

File: OpenOversight/app/templates/partials/officer_incidents.html [hunk 1]
```diff
@@ -2,7 +2,7 @@ <h3>Incidents</h3>
 {% if officer.incidents %}
   <table class="table table-hover table-responsive">
     <tbody>
-      {% for incident in officer.incidents | sort(attribute='date') | reverse %}
+      {% for incident in officer.incidents %}
```
Comment (michplunkett) [inline OpenOversight/app/templates/partials/officer_incidents.html:5]: This is now covered in the view.

File: OpenOversight/app/models/database_imports.py [hunk 2]
```diff
@@ -223,12 +240,12 @@ def update_link_from_dict(data: Dict[str, Any], link: Link) -> Link:
         link.description = parse_str(data.get("description"), None)
     if "author" in data:
         link.author = parse_str(data.get("author"), None)
-    if "created_by" in data:
-        link.created_by = parse_int(data.get("created_by"))
```
Comment (michplunkett) [inline OpenOversight/app/models/database_imports.py:227]: The `created_by` field shouldn't be updated.

**Review by sea-kelp — APPROVED** [review]
Should this also be replaced with a constant?
https://github.com/lucyparsons/OpenOversight/pull/1056/files#diff-0ea02af7be8529b5f1eb21efd1af13f1d633060c50ee2529a1064ddba1616c93L76-L80

**Comment by michplunkett** [comment]
@sea-kelp, at the very least it should be an f-string. We can change that on the next code change.

**Pull request merged**